src/scripts/main.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const body = document.body;
+
+  body.insertAdjacentHTML('afterbegin', `
+    <div class="notification ${type}" style=top: ${top}px; left: ${right}px;>
+      <h2 class="title">${title}</h2>
+      <p>${description}</p>
+    </div>
+  `);
+
+  setTimeout(() => document.querySelector('div').remove(), 2000);
```

Comment:
should we show a message after 2s or delete?

Reply:
delete